programs/benchzstd.c

```diff
@@ -431,10 +429,9 @@ BMK_benchMemAdvancedNoAlloc(
         dctxprep.dictBuffer = dictBuffer;
         dctxprep.dictBufferSize = dictBufferSize;
 
-        hr_isize = UTIL_makeHumanReadableSize((U64) srcSize);
-
         DISPLAYLEVEL(2, "\r%70s\r", "");   /* blank line */
-        DISPLAYLEVEL(2, "%2s-%-17.17s : %.*f%s -> \r", marks[markNb], displayName, hr_isize.precision, hr_isize.value, hr_isize.suffix);
+        assert(srcSize < UINT_MAX);
+        DISPLAYLEVEL(2, "%2s-%-17.17s :%10u -> \r", marks[markNb], displayName, (unsigned)srcSize);
```

Comment:
With the existing code, you can see full sizes with `-vv`. Does that seem insufficient? In general I like that this is human-readable formatted.

Reply:
Ah yes, I noticed this capability.
It wasn't enough unfortunately, as it was breaking scripts depending on previous format.
Moreover, I simply believe that this is not the right place for the condensed quantity format. Benchmarking is about accuracy. I typically always need the exact amount, in order to clearly appreciate changes, even when they are small.

Reply:
Yeah, I will say more often than not I have found myself benchmarking something and then having to re-do it once i realize I forgot `-vv`. At least for me, I'd always prefer to have it on.

Reply:
Makes sense.